src/test/java/uk/gov/hmcts/reform/civil/utils/CaseFlagsToHearingValueMapperTest.java

```diff
@@ -68,24 +69,142 @@ public void testHasAdditionalSecurityFlag() {
         assertEquals(true, getAdditionalSecurity(flagDetails));
     }
 
-    @Test
-    public void testHasLanguageInterpreterFlag() {
-        FlagDetail flagDetail1 = FlagDetail.builder()
-            .status("Active")
-            .hearingRelevant(YES)
-            .flagCode("PF0015")
-            .subTypeValue("english")
-            .build();
-
-        FlagDetail flagDetail2 = FlagDetail.builder()
-            .status("INACTIVE")
-            .hearingRelevant(YES)
-            .flagCode("PF0015")
-            .build();
-
-        List<FlagDetail> flagDetails = List.of(flagDetail1, flagDetail2);
+    @Nested
+    class GetLanguageInterpreter {
+        @Test
```

Comment:
Can we add a test for when both Sign Language and Spoken Language keys are set to show the preferrence for interpreterLanguage field is the Spoken language when both are set?

Reply:
I added it to shouldReturnFirstSignLanguageInterpreterFlag() (line 181)